### ISSUE TITLE
Islands not detected by prebundle check or qmos 'Color Islands'

### ISSUE THREAD

**Issue by ladoramkershner** [DOI-USGS/ISIS3#3983]
**ISIS version(s) affected**: x.y.z
Noticed on 4.0.1, did not check other versions.

**Description**  
A network passes the jigsaw prebundle check and shows no islands in qmos (when using 'Color Islands' button), but fails cnetcheck with two distinct islands.

**How to reproduce**  
Take the following list of images and network
/scratch/ladoramkershner/mars_quads/oxia_palus/mc11_ns8_ew1_t121x61_active.lis
/scratch/ladoramkershner/mars_quads/oxia_palus/mc11_ns8_ew1_t121x61_active.net
and open with qmos, press the 'Color Islands' button, cubes should remain multicolored and placed in a single group. Additionally run jigsaw, the bundle is not expected to converge but validation will complete. Finally, run cnetcheck, the network should have two Islands.

**Possible Solution**  

**Additional context**

**Comment by lwellerastro** [comment]
@ladoramkershner jigsaw will bundle a network that contains islands.  How well it does so will likely depend on the data and what is being solved for.  I was able to bundle 482 Galileo SSI images that had 28 distinct islands without a problem.  I didn't use the output for anything other than a general evaluation of things since the intent was to deal with each island in and of itself and get them all reconnected again. 

The only pre-bundle check I know of that jigsaw performs is checking for images with one point.

Qmos on the other hand is a little concerning.  I have used the color islands function many times with success so that one is odd.

I checked the output of cnetcheck in your directory and did a count on the islands files:
```
wc -l /scratch/ladoramkershner/mars_quads/oxia_palus/ACTIVE_CHECK*Island*
  3326 /scratch/ladoramkershner/mars_quads/oxia_palus/ACTIVE_CHECKIsland.1
     2 /scratch/ladoramkershner/mars_quads/oxia_palus/ACTIVE_CHECKIsland.2
```
My guess is the 2 images in island 2 are being hidden by the other 3326 footprints and the default color it chose is not helping.  You can use the filename search box under FileList and look for the two images in the island 2 list. Right click the highlighted name in the list and select zoom fit to more or less get you there.  Also, if you toggle back and forth between Color Islands and Color Images you might get a better color selection that helps them stand out.

FYI - jigsaw will definitely bundle this sort of situation as long as the network is in ok shape.  I had a two image island in my global Themis network for a long time before I ran cnetcheck and found it (highly recommend running this before jigsaw and if a lot of clean up is occurring - don't get lazy like I did; you can always set latlon=false to make it go quicker for data without limb).  Jigsaw simply did not care.

**Comment by ladoramkershner** [comment]
@lwellerastro Thank you for the response. I had a feeling jigsaw did not do a thorough check of the network, but I didn't want to dig through the code to figure out what it was doing. The precheck+bundle process can already lengthy and since island do not prevent the solution but single measure points do, it makes sense that would be the check it prioritizes. 

I cycled the 'Color Images' and 'Color Islands' this morning and it is highlighting the islands, but previously all the cubes were all different colors. I am assuming I messed up something, but ISIS is definitely fine. Sigh..
